source/macos/AdaptiveCards/AdaptiveCards/Views/ACRTextField.swift

```diff
@@ -15,40 +15,28 @@ class ACRTextField: NSTextField {
     weak var textFieldDelegate: ACRTextFieldDelegate?
     private let config: RenderConfig
     private let inputConfig: InputFieldConfig
+    private let inputElement: ACSBaseInputElement?
     private let isDarkMode: Bool
     private let textFieldMode: Mode
     private var shouldShowError = false
     private var errorMessage: String?
     private var labelString: String?
     
-    private init(with config: RenderConfig, mode: Mode, inputElement: ACSBaseInputElement?) {
+    init(textFieldWith config: RenderConfig, mode: Mode, inputElement: ACSBaseInputElement?) {
         self.config = config
```

Comment:
```suggestion
    init(textFieldWith config: RenderConfig, mode: Mode, inputElement: ACSBaseInputElement) {
        self.config = config
```

Should be **non-optional**